flash/vision/classification/data.py

```diff
@@ -387,12 +410,15 @@ def from_folders(
             train/cat/asd932.png
 
         Args:
-            train_folder: Path to training folder.
-            train_transform: Image transform to use for training set.
-            valid_folder: Path to validation folder.
+            train_folder: Path to training folder. Default: None.
+            valid_folder: Path to validation folder. Default: None.
+            test_folder: Path to test folder. Default: None.
+            predict_folder: Path to predict folder. Default: None.
```

Comment:
```suggestion
            train_folder: Path to training folder.
            valid_folder: Path to validation folder.
            test_folder: Path to test folder.
            predict_folder: Path to predict folder.
```
no need to repeat as it is already in func signature

Reply:
It think it is fine to have the defaults at the end.